tox.ini

```diff
@@ -212,10 +212,15 @@ commands =
     bash ./scripts/check_file_names.sh
     python ./scripts/check_asf_license_headers.py .
 
-[testenv:integration]
-deps = -r{toxinidir}/integration/requirements.txt
+[testenv:integration-compute]
+deps = -r{toxinidir}/integration/compute/requirements.txt
 
-commands = python -m integration
+commands = python -m integration.compute
+
+[testenv:integration-storage]
+deps = -r{toxinidir}/integration/storage/requirements.txt
+
+commands = python -m integration.storage
```

Comment:
This is fine for now, although in the future I would prefer to use ``pytest`` directly - this way common pytest flags (e.g. ``-s``) and plugins can be used.

Reply:
EDIT: Sorry, I just see you followed an existing pattern since original compute integration tests code is quite old :)